robusta/src/main.rs

```diff
@@ -267,6 +270,13 @@ async fn main() {
     info!("Starting server");
     kvv::init().await;
 
+    let specta_emit_path =
+        std::env::var("SPECTA_EMIT_PATH").unwrap_or("../liberica/src/lib/bindings.ts".to_string());
+
+    if let Err(err) = specta::export::ts(&specta_emit_path) {
+        error!("Could not emit TS types to '{specta_emit_path}': {err}")
+    };
+
```

Comment:
what does this do / why does it exist? This unconditionally overwrites the file prompting a frontend rebuild even if nothing has changed, for reference, read the lines above